docs/tutorials/tutorial_with_repositoryserver.rst

```diff
@@ -0,0 +1,538 @@
+Using Kopia Repository Server as Data Mover in Blueprint
+********************************************************
+
+This tutorial will demonstrate the use of Kopia to copy/restore backups
+to a Kopia repository. We will be using kanister functions
+that use Kopia repository Server as data mover in the blueprint.
+For additional documentation on kanister functions and blueprints
+refer to the :ref:`architecture` and :ref:`kanister functions<functions>`
+sections respectively
+
+.. contents:: Tutorial Overview
+  :local:
+
+Prerequisites
+=============
+
+* Kubernetes ``1.16`` or higher. For cluster version lower than ``1.16``,
+  we recommend installing Kanister version ``0.62.0`` or lower.
+
+* `kubectl <https://kubernetes.io/docs/tasks/tools/install-kubectl/>`_ installed
+  and setup
+
+* `helm <https://helm.sh>`_ installed and initialized using the command `helm init`
+
+* docker
+
+* Kopia repository server controller should be deployed along with Kanister controller
+  Refer
+  :ref:`Deploying Kopia Repository server controller <deploying_repo_server_controller>`
+
+* Access to s3 bucket and credentials
+
+Example Application
+===================
+
+This tutorial begins by deploying a sample application. The application is
+contrived, but useful for demonstrating Kanister's features. The application
+appends the current time to a log file every second. The application's container
+includes the aws command-line client which we'll use later in the tutorial. The
+application is installed in the ``default`` namespace.
```

Comment:
```suggestion
This tutorial begins by deploying a sample application. While this application is
intentionally simplified, it serves the purpose of demonstrating Kanister's features. The application
appends the current time to a log file every second. The application's container
includes the aws command-line client, which we will use in later sections of this tutorial. This
application is installed within the ``default`` namespace.
```

We can use words such as "designed" or "crafted" or "created" instead of the word "contrived".